app/styles/ui/_blank-slate.scss

```diff
@@ -1,105 +1,171 @@
 #blank-slate {
   display: flex;
-  padding: var(--spacing);
+  padding: calc(var(--spacing) * 6);
   align-items: center;
 
-  .header {
-    max-width: 400px;
-    align-self: center;
+  header {
+    align-self: flex-start;
+    margin-bottom: var(--spacing-quad);
 
-    text-align: center;
-  }
+    h1 {
+      font-size: var(--font-size-xxl);
+      font-weight: var(--font-weight-light);
+    }
 
-  .title {
-    font-size: var(--font-size-md);
-    font-weight: var(--font-weight-semibold);
-    text-align: center;
-    margin-bottom: var(--spacing-double);
+    h1,
+    p {
+      margin: 0;
+    }
   }
 
-  .content,
-  .footer {
+  .content {
+    align-self: stretch;
+    flex-direction: row;
     display: flex;
-    flex-direction: column;
-    align-self: center;
     flex-shrink: 0;
-  }
+    flex-grow: 1;
 
-  .content {
-    margin: var(--spacing-double) 0;
-    min-width: 80%;
-  }
+    & > .content-pane {
+      display: flex;
+      flex-direction: column;
+      align-self: stretch;
+      flex-shrink: 0;
+      max-width: 50%;
+      width: 50%;
+      flex-grow: 1;
+
+      &:not(:last-child) {
+        padding-right: var(--spacing);
+      }
 
-  .footer {
-    margin: 0;
+      &:not(:first-child) {
+        padding-left: var(--spacing);
+      }
+    }
+
+    ul.button-group {
+      list-style-type: none;
+      margin: 0;
+      padding: 0;
+      flex-grow: 1;
+
+      li {
+        margin-bottom: var(--spacing);
+        width: 100%;
+
+        .button-component {
+          width: 100%;
+          padding: var(--spacing);
+          height: auto;
+          text-align: left;
+          margin-bottom: var(--spacing);
+          display: flex;
+          flex-direction: row;
+          align-items: center;
+
+          /** These buttons need to wrap in high zoom levels */
+          text-overflow: unset;
+          white-space: normal;
+          min-width: 0;
+
+          .octicon {
+            margin-right: var(--spacing);
+            height: 32px;
+            width: 32px;
+          }
+        }
+      }
+    }
   }
 
-  .callouts {
+  .drag-drop-info {
+    background: var(--tip-box-background-color);
+    border: 2px dashed var(--tip-box-border-color);
+    border-radius: var(--border-radius);
+    padding: var(--spacing-double);
     display: flex;
-    flex-shrink: 0;
-    min-width: 0;
-    min-height: 0;
-    justify-content: center;
-  }
+    flex-direction: row;
 
-  .callout {
-    &:last-child {
-      padding-right: 0;
+    .octicon {
+      margin-right: var(--spacing);
     }
+  }
 
-    &:first-child {
-      padding-left: 0;
-    }
+  .blankslate-graphic-bottom {
+    position: absolute;
+    right: var(--spacing);
+    bottom: var(--spacing);
 
-    display: flex;
-    flex-direction: column;
-    flex-shrink: 0;
+    width: auto;
+    max-width: 40%;
+    height: 40%;
+    // I hate this but we'll have to live with it for beta.
```

Comment:
What's the TL;DR on this hack? Is it worth capturing as an issue?

Reply:
This is duplicated from the welcome styles. The hack is that we're using z-index which I don't like but I don't think it's worth worrying about. I can remove the comment from here